deployment/elasticsearch/elasticsearch.go

```diff
@@ -243,7 +243,7 @@ func (r RestoreSnapshotOpts) MarshalJSON() ([]byte, error) {
 	}
 
 	// Set the number of replicas of each shard
-	numReplicas := 1
+	numReplicas := 0
```

Comment:
Why not use `r.NumberOfReplicas` directly in the first place?

Reply:
In case the caller set a negative number. But in this case we should throw an error, so I added a small validation step before. See https://github.com/mattermost/mattermost-load-test-ng/pull/764/commits/32788e6abf8395768afb2d189a89b58b1e9fd5b1

Reply:
Why not make it unsigned, then? Just kidding, it's the endless dilemma :stuck_out_tongue: 

I like the new approach, failing fast in case of a clear input error.